pokemon.test.js

```diff
@@ -0,0 +1,145 @@
+require('dotenv').config();
+const fakeRequest = require('supertest');
+const index = require('./index');
+const { execSync } = require('child_process');
+const client = require('./lib/utils/pool');
+
+describe('test Pokemon model', () => {
+
+  // need to manually run 'npm run drop-tables' then 'npm run create tables' currently
+  // can't figure out how to run these scripts. They work in command line...
+  // beforeAll(async done => {
+  //   execSync('npm run setup-tables');
+  //   client.connect();
+  //   return done();
+  // });
+
+  // afterAll(done => {
+  //   return client.end(done);
+  // });
+
+
+  it('insert Pokemon Moltres', async() => {
```

Comment:
seems like these tests are not passing CI.  When you can, I would check why CI is not passing, and try to get these tests to pass for those final points.